linear_operator/test/linear_operator_test_case.py

```diff
@@ -1247,3 +1247,14 @@ def test_svd(self):
             for arg, arg_copy in zip(linear_op.representation(), linear_op_copy.representation()):
                 if arg_copy.requires_grad and arg_copy.is_leaf and arg_copy.grad is not None:
                     self.assertAllClose(arg.grad, arg_copy.grad, **self.tolerances["svd"])
+
+    def test_to_double(self):
+        # test if the linear_op is still functional and converted to torch.float64 after calling to(torch.float64).
+        linear_op = self.create_linear_op()
+        try:
+            linear_op = linear_op.to(torch.float64)
+            linear_op.numpy()
+        except RuntimeError:
+            raise TypeError(f"Could not convert {type(linear_op)} to double.")
```

Comment:
This seems more like a `RuntimeError` than like a `TypeError`
```suggestion
            raise RuntimeError(f"Could not convert {type(linear_op)} to double.")
```